test/function_test.py

```diff
@@ -972,3 +972,24 @@ def f(x):
         @batched(max_batch_size=1, wait_ms=1)
         def g(x=1):
             return [x_i**2 for x_i in x]
+
+
+@pytest.mark.parametrize("feature_flag", [True, False, None])
+def test_spawn_extended_feature_flag(client, servicer, monkeypatch, feature_flag):
+    app = App()
+    dummy_modal = app.function()(dummy)
+
+    if feature_flag:
```

Comment:
```suggestion
    if feature_flag is not None:
```

I think you want to excise "set to True, set to False, and not set" here but I may be wrong.